RDBM/Table.cpp

```diff
@@ -193,18 +211,106 @@ void Table::Show(std::ostream &out)
 		ptr = table[i];
 		ptr->Show(out);
 	}
+	return *this;
 }
 
 
 
-void Table::Delete()
+Table& Table::Delete()
```

Comment:
Why do we need to return table here?

Reply:
delete is working like a clean function,it deletes all the data inside table ,but not the table itself.so you could use Delete to clean the table and then use it as a new one without creating new table.
maybe Table.Clear() would be a better name

Reply:
I understand but this way of this function gives us possibility to write "Table.Clear().Add(some_data);"
Will we use it in the future?

Reply:
why not?